### PR TITLE
Add view_health_history_requests API call

### PR DESCRIPTION
**Pull request by sarthakgoel1997**
## Description
- Added **view_health_history_requests** API call
- The implementation is similar **view_health_history** backend call, hence moved the common part to a `utils.py` file
- Mapped a `record_status` field in the common util

## How did you test the changes?
- Tested the changes by checking the response from common util on **View Health History** page

## Testing Screenshots (Optional)

image

### CODE REVIEW

**Review by gamSeier — APPROVED** [review]

File: healthScore/utils.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+from .models import (
+    healthRecord,
+    hospital,
+    hospitalStaff,
+    appointment,
+)
+
+
+def get_health_history_details(request):
```
Comment (shreyas2499) [inline healthScore/utils.py]: We can have this in views.py itself right?

File: healthScore/views.py [hunk 2]
```diff
@@ -284,7 +212,7 @@ def registration(request):
             )
             return render(request, "registration.html", context)
 
-        elif user.objects.filter(userName=username).exists():
+        elif user.objects.filter(username=username).exists():
```
Comment (shreyas2499) [inline healthScore/views.py:215]: Won't these have to be userName? Since filter will search for the userName key in the db?

Comment (gamSeier) [reply healthScore/views.py:215]: Sarthak adjusted the DB column header to be "username" to deal with an issue I was running into

Comment (shreyas2499) [reply healthScore/views.py:215]: Ohh, alright

File: healthScore/utils.py (hunk 1 above)
Comment (sarthakgoel1997) [reply healthScore/utils.py]: Yeah right. I just want to keep reusable utility functions in a separate file and don't want to make the `views.py` file too long. Should I move it to `views.py`?

Comment (shreyas2499) [reply healthScore/utils.py]: Oh, that makes sense. We can maybe keep the files like role specific (unless features overlap). 
Like healthCare workers could have like a healthCareViews.py or things like that?

Comment (sarthakgoel1997) [reply healthScore/utils.py]: Sure, we can do that. Should I name the file `user_utils.py`?

Comment (shreyas2499) [reply healthScore/utils.py]: Yeah, I think that should be good then. Approving the changes

**Review by shreyas2499 — APPROVED** [review]

File: healthScore/utils.py (hunk 1 above)
Comment (sarthakgoel1997) [reply healthScore/utils.py]: Done, thanks!

**Review by shreyas2499 — APPROVED** [review]

**Pull request merged**